go/mysql/flavor_mysql.go

```diff
@@ -32,7 +32,8 @@ type mysqlFlavor struct{}
 
 // masterGTIDSet is part of the Flavor interface.
 func (mysqlFlavor) masterGTIDSet(c *Conn) (GTIDSet, error) {
-	qr, err := c.ExecuteFetch("SELECT @@GLOBAL.gtid_executed", 1, false)
+	// making @@global as lowercase, as the PITR depends on binlog server, which honours only lowercase `global` value
```

Comment:
Let's just make this:

```
// keep @@global as lowercase, as some servers like the Ripple binlog server only honors a lowercase `global` value
```

Otherwise people might get confused (since this is not PITR-specific code)

Reply:
Agreed, I have updated the comment.